src/styles/body.scss

```diff
@@ -0,0 +1,9 @@
+
+body {
+  margin: 0;
+  height: 100vh;
```

Comment:
It's not recommended to set a fixed height on the `<body>` element. This can lead to unexpected behavior when the content exceeds the height of the viewport. Consider using `min-height` instead to ensure that the body expands with the content.